.github/main.workflow

```diff
@@ -0,0 +1,84 @@
+workflow "Build and Test" {
+  on = "push"
+  resolves = ["Test", "LintersOK"]
+}
+
+action "Docker Registry Login" {
+  uses = "actions/docker/login@master"
+  secrets = ["DOCKER_USERNAME", "DOCKER_PASSWORD"]
+}
+
+action "Cached Build" {
+  uses = "actions/docker/cli@master"
+  runs = ["sh", "-c", "BRANCH=${GITHUB_REF##*/} ID=gh ci/build.sh"]
+  needs = ["Docker Registry Login"]
+  env = {
+    PUSH_CACHE = "0"
+  }
+}
+
+action "Caching Build" {
+  uses = "actions/docker/cli@master"
+  runs = ["sh", "-c", "BRANCH=${GITHUB_REF##*/} ID=gh ci/build.sh"]
```

Comment:
should we get rid of the part after `REF` or is this saved here to be uncommented later? (same with the one above?)

Reply:
Weirdly this is a bug in GitHub web interface's syntax highlighting.  Those hashmarks are part of a command transforming the variable to strip out everything prior to the the final slash, in order to extract just the branch name. (Isn't bash so clear and easy to read? Ugh.)